src/ITileSqliteDatabasesCollection.cpp

```diff
@@ -0,0 +1,9 @@
+#include "ITileSqliteDatabasesCollection.h"
+
+OsmAnd::ITileSqliteDatabasesCollection::ITileSqliteDatabasesCollection()
```

Comment:
Suggestion:
We can use modern style `default` keyword  here or in the header.
Like this:
```
OsmAnd::ITileSqliteDatabasesCollection::ITileSqliteDatabasesCollection() = default;
```

Reply:
Thanks for the suggestion, yet until since this PR is not about change of style, I see no reason changing this only here. On top of that, until it's possible to verify new approach on all build flavours and configurations (because there's no CI that does it) - I'll avoid introducing new approach unless that's justified. Also, I'm not sure how this would play along with memory tracing during runtime (that is, who's memory memory management is going to be used - host's or library's) - additional testing and check-up on specs is required prior to blindly adopting new style.